udata/core/spatial/models.py

```diff
@@ -24,6 +24,9 @@
 
 ADMIN_LEVEL_MIN = 1
 ADMIN_LEVEL_MAX = 110
+# Max known population for a zone
+# World population is 6772425850.
+POPULATION_MAX = 2147483647
```

Comment:
I think it comes from an ES limitation given that it's [the maximum positive value for a 32-bit signed binary integer](https://en.wikipedia.org/wiki/2147483647_(number)). So it's not related to the population.